test/signalrclient-e2e-tests/url.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+extern utility::string_t url;
```

Comment:
The variable url only can be defined one time, it is defined in test_utils.h.
In url.h the variable url is defined as extern which tells the compiler that the variable is defined somewhere else, so connection_tests file and hub_connection_tests both can include url.h.